modules/virt-configuring-configmap-for-obsolete-cpu-models.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * virt/node_maintenance/virt-managing-node-labeling-obsolete-cpu-models.adoc
+
+[id="virt-configuring-configmap-for-obsolete-cpu-models_{context}"]
+= Configuring a ConfigMap for obsolete CPU models
+
+Use this procedure to configure a ConfigMap for obsolete CPU models.
```

Comment:
Since it's a nit, leaving it stet - we use the intro sentence "Use this procedure..." in our OpenShift Virt docs. One example - 
https://docs.openshift.com/container-platform/4.5/virt/logging_events_monitoring/virt-monitoring-vm-health.html